pwiz_tools/Skyline/TestTutorial/TargetedMSMSTutorialTest.cs

```diff
@@ -1188,6 +1199,76 @@ private void TestPropertySheet()
             OkDialog(dlg, () => dlg.Close());
         }
 
+        private void TestLibraryMatchPropertySheet()
+        {
+            var isSmallMolecules = AsSmallMoleculesTestMode != RefinementSettings.ConvertToSmallMoleculesMode.none;
+
+            var expectedPropertiesJson =
+                    "{  \"IdFileName\": \"klc_20100329v_Protea_Peptide_Curve_20fmol_uL_tech1.perc.xml\",  \"FileName\": \"klc_20100329v_Protea_Peptide_Curve_20fmol_uL_tech1.ms2\",  \"FilePath\": \"C:\\\\Users\\\\RitaCh\\\\Workspaces\\\\ProteoWiz\\\\pwiz2\\\\pwiz_tools\\\\Skyline\\\\TestResults\\\\TargetedMSMSMzml_2\\\\TargetedMSMSMzml\\\\Low Res\\\\search\\\\klc_20100329v_Protea_Peptide_Curve_20fmol_uL_tech1.ms2\",  \"LibraryName\": \"BSA_Protea_label_free_meth3\",  \"PrecursorMz\": \"582.319\",  \"Charge\": 2,  \"Label\": \"light\",  \"RetentionTime\": \"46.81\", \"CCS\": null,  \"IonMobility\": null,  \"SpecIdInFile\": \"7901\",  \"Score\": 0.0,  \"ScoreType\": \"Percolator q-value\",  \"SpectrumCount\": 118}";

```

Comment:
This is kind of a mess. Please use consistent ordering between the two cases. Ideally a format that is not quite so heavy on backslashes, if possible, and it looks like this would need a record mode in case anything changes. Look at other tests that do this. I am not clear on why small molecule mode does not preserve things like FileName. Please at least clarify that with a comment.

Reply:
Fixed the test data format. The missing file name was a bug, I fixed that too. Waiting for the tests to pass before merging.